verilog/tools/kythe/indexing_facts_tree.h

```diff
@@ -30,44 +30,34 @@
 namespace verilog {
 namespace kythe {
 
-// Anchor class represents the location and value of some token.
-class Anchor {
-  // Disable implicit conversions when passing values to overloaded
-  // function/constructor.  This work by virtue of the language only allowing
-  // one implicit conversion at most.
-  // e.g. some_function(DisableConversion<absl::string_view>)
-  // accepts only string_views, while disabling string_view's implicit
-  // conversion from std::string (and others).
-  template <class T>
-  struct DisableConversion {
-    /* implicit */ DisableConversion(T v) : value(v) {}  // NOLINT
+// Position of the Anchor in the original text.
+struct AnchorOffset {
+  const int begin;
```

Comment:
It feels like these want to be `size_t` maybe (I guess it depends a bit what the kythe protocol buffer size of this allows). 

Given that this is representing a begin + length, wouldn't this be, naming-wise, more an `AnchorRange` or `AnchorSpan` ?

It might be worthwhile documenting why this class exists and is not simply a string_view into the larger content (because string might not exist anymore), so capturing the file offset and length is needed.

Reply:
All done. AnchorRange fits much better than Offset. Thanks!